### ISSUE TITLE
Export filename ends with '.html' when using web ui

### ISSUE THREAD

**Issue by siven-xtn** [FriendsOfSylius/SyliusImportExportPlugin#165]
When exporting from the web UI, the export filename ends with '.html'. 

Issue found on Sylius 1.3.7

The solution is to add the 'Content-Type' header to the response.

A simple fix (but not the right one) is to add the following after line 80 in the file vendor/friendsofsylius/sylius-import-export-plugin/src/Controller/ExportDataController.php
```
$response->headers->set('Content-Type', 'application/' . $format);
```

**Comment by alexweggs** [comment]
Weirdly, i have this bug now but was working fine til today !

I confirmed if you had the content-type header it works fine.

**Comment by oallain** [comment]
Resolved by #223
